unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -335,7 +335,7 @@ async UniTaskVoid LoadUserProfileAsync(UserProfile userProfile, CancellationToke
 
                     for (var i = 0; i < wearablesCount; i++)
                     {
-                        string wearableId = userProfile.avatar.wearables[i];
+                        string wearableId = new UrnAndTokenId(userProfile.avatar.wearables[i]).Urn;
```

Comment:
Instead of allocating a temporary object for solving the urn, its preferible to make a static call or extension method